tools/dataProtector/dataProtectorCore/processProtectedData.md

```diff
@@ -143,6 +143,60 @@ const processProtectedDataResponse =
   });
 ```
 
+### useVoucher <OptionalBadge />
+
+**Type:** `boolean`  
+**Default:** `false`
+
+This optional param allows you to pay for the deal using your voucher. Make sure
+that your voucher is held by your connected wallet.
```

Comment:
- "deal" is not mentioned in other payment-related options (ie: `maxPrice`); instead, "task" is used, we should keep it uniformized.
- the voucher used is not always the user's voucher, maybe say it uses user's voucher by default and can be override by `voucherAddress`